### PR TITLE
section 2 for cis fixes

### PR DESCRIPTION
**Pull request by fdistorted**

### CODE REVIEW

File: resources/resources_policy_assignments.go [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+			},
+			{
+				Name:        "not_scopes",
+				Description: "The policy's excluded scopes",
+				Type:        schema.TypeStringArray,
+				Resolver:    schema.PathResolver("AssignmentProperties.NotScopes"),
+			},
+			{
+				Name:        "parameters",
+				Description: "The parameter values for the assigned policy rule The keys are the parameter names",
```
Comment (irmatov) [inline resources/resources_policy_assignments.go:54]: nit: description should be shortened

File: resources/resources_policy_assignments.go [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+			},
+			{
+				Name:        "description",
+				Description: "This message will be part of response in case of policy violation",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("AssignmentProperties.Description"),
+			},
+			{
+				Name:        "metadata",
+				Description: "The policy assignment metadata Metadata is an open ended object and is typically a collection of key value pairs",
```
Comment (irmatov) [inline resources/resources_policy_assignments.go:66]: nit: description should be shortened

File: resources/resources_policy_assignments.go [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+				Type:        schema.TypeJSON,
+				Resolver:    resolveResourcesPolicyAssignmentMetadata,
+			},
+			{
+				Name:        "enforcement_mode",
+				Description: "The policy assignment enforcement mode Possible values are Default and DoNotEnforce Possible values include: 'Default', 'DoNotEnforce'",
```
Comment (irmatov) [inline resources/resources_policy_assignments.go:72]: nit: description should be shortened

File: resources/resources_policy_assignments.go [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "name",
+				Description: "The name of the policy assignment",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "sku_name",
+				Description: "The name of the policy sku Possible values are A0 and A1",
```
Comment (irmatov) [inline resources/resources_policy_assignments.go:94]: nit: description should be shortened

File: resources/resources_policy_assignments.go [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("Sku.Name"),
+			},
+			{
+				Name:        "sku_tier",
+				Description: "The policy sku tier Possible values are Free and Standard",
```
Comment (irmatov) [inline resources/resources_policy_assignments.go:100]: nit: description should be shortened

File: resources/resources_policy_assignments.go [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("Sku.Tier"),
+			},
+			{
+				Name:        "location",
+				Description: "The location of the policy assignment Only required when utilizing managed identity",
```
Comment (irmatov) [inline resources/resources_policy_assignments.go:106]: nit: description should be shortened

File: resources/resources_policy_assignments.go [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+			},
+			{
+				Name:        "identity_tenant_id",
+				Description: "The tenant ID of the resource identity",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("Identity.TenantID"),
+			},
+			{
+				Name:        "identity_type",
+				Description: "The identity type This is the only required field when adding a system assigned identity to a resource Possible values include: 'SystemAssigned', 'None'",
```
Comment (irmatov) [inline resources/resources_policy_assignments.go:123]: nit: description should be shortened

**Review by irmatov — APPROVED** [review]
LGTM. It would be nice to adjust the descriptions.

**Review by roneli — APPROVED** [review]
LGTM, fix @irmatov comments and merge

**Pull request merged**